lib/beaker-answers/answers.rb

```diff
@@ -55,32 +76,42 @@ def self.supported_versions
     # @param [Array<Beaker::Host>] hosts An array of host objects.
     # @param [Hash] options options for answer files
     # @option options [Symbol] :type Should be one of :upgrade or :install.
+    # @option type [Symbol] :format Should be one of :bash or :hiera
+    #   Setting :bash will result in the "classic" PE answer file being generated
+    #   Setting :hiera will generate the new PE hiera config file format
     # @return [Hash] A hash (keyed from hosts) containing hashes of answer file
     #   data.
-    def self.create version, hosts, options
+    def self.create(version, hosts, options, type = :bash)
```

Comment:
Options is a Hash, type is a param being added to the call sig.  The comment above is listing it as an @option element of the options Hash.  That might (?) be the better way to pass this though, rather than adding a parameter?  Although ultimately the type should be based on which version class is selected.  However we do currently need a hybrid class than can work either way.  Perhaps rather than a parameter, set @type = :bash in the Answers#initialize() and override from options; still pondering this.